VimeoNetworking/VimeoNetworking/Models/VIMVideo.m

```diff
@@ -139,6 +139,34 @@ - (void)didFinishMapping
     [self setVideoStatus];
 }
 
+#pragma mark - Model Validation
+
+- (void)validateModel:(NSError *__autoreleasing *)error
+{
+    [super validateModel:error];
+    
+    if (*error)
+    {
+        return;
+    }
+    
+    if (self.uri == nil)
+    {
+        NSString *description = @"VIMVideo failed validation: uri cannot be nil";
+        *error = [NSError errorWithDomain:VIMModelObjectErrorDomain code:VIMModelObjectValidationErrorCode userInfo:@{NSLocalizedDescriptionKey: description}];
+        
+        return;
+    }
+    
+    if (self.resourceKey == nil)
```

Comment:
tomato tomahto, only matters if both are nil, and I feel like if both are nil we're in deep 💩